index.js

```diff
@@ -531,6 +531,28 @@ exports.generateAddress = function (from, nonce) {
   return exports.rlphash([from, nonce]).slice(-20)
 }
 
+/**
+ * Generates an address for a contract created using CREATE2
+ * @param {Buffer} from the address which is creating this new address
+ * @param {Buffer} salt a salt
+ * @param {Buffer} initCode the init code of the contract being created
+ * @return {Buffer}
+ */
+exports.generateAddress2 = function (from, salt, initCode) {
+  from = exports.toBuffer(from)
+  salt = exports.toBuffer(salt)
```

Comment:
Should the length of the buffers be checked (as they should be 20 and 32, else the hashes would differ, right?)